pkg/awsapi/generate/add_import.go

```diff
@@ -0,0 +1,73 @@
+//go:build ignore
```

Comment:
@cPu1 Could this be separate from this PR perhaps? It's obscure enough to have it's own improvement. I like using the Go AST, but this requires a bit of testing and documentation on why it exists and what it does.

Reply:
Never mind. I was convinced otherwise.